### PR TITLE
Add new command "pano field cleanup"

### PR DESCRIPTION
**Pull request by mrknmc**
Discussed the naming with @kubamahnert 

* Looks for dataset-scoped field files with no calculation defined
* Output also included in `pano validate`

Planning to add a command for generating field files from field_map in model files and will make a new release

### CODE REVIEW

File: src/panoramic/cli/cli.py [hunk 1]
```diff
@@ -183,5 +183,19 @@ def validate():
 def detect_joins(target_dataset: str, yes: bool, diff: bool, overwrite: bool):
     from panoramic.cli.command import detect_joins as detect_joins_command
 
-    if not detect_joins_command(target_dataset=target_dataset, diff=diff, overwrite=overwrite, yes=yes):
-        sys.exit(1)
+    detect_joins_command(target_dataset=target_dataset, diff=diff, overwrite=overwrite, yes=yes)
+
+
+@cli.group(name='field')
```
Comment (raethlo) [inline src/panoramic/cli/cli.py:189]: [offtopic] Just a thought. We will be adding a couple of groups in the coming releases - might be a good idea to slowly start organising them into separate modules.

File: src/panoramic/cli/command.py [hunk 2]
```diff
@@ -291,6 +288,7 @@ def detect_joins(target_dataset: Optional[str] = None, diff: bool = False, overw
 
     if local_state.is_empty:
         echo_info('No datasets to detect joins on')
+        # TODO: should we return here?
```
Comment (raethlo) [inline src/panoramic/cli/command.py:291]: Yeah, good spot - this is missing a `return` for sure - I think we can add it and if it doesn't break tests we should be good to go.

**Review by raethlo — APPROVED** [review]
Nice! Left a few minor comments apart from what we discussed before.

File: src/panoramic/cli/command.py (hunk 2 above)
Comment (mrknmc) [reply src/panoramic/cli/command.py]: Added the return in 07d5fd42df578ce45b4630276102c7e926fc8559

**Pull request merged**